aframe/examples/marker-events.html

```diff
@@ -36,7 +36,7 @@
 	<br/>
 	Contact me any time at <a href='https://twitter.com/jerome_etienne' target='_blank'>@jerome_etienne</a>
 </div>
-	<a-scene embedded arjs='sourceType: webcam; detectionMode: mono_and_matrix; matrixCodeType: 3x3; sourceWidth:1280; sourceHeight:960; displayWidth: 1280; displayHeight: 960;'>
```

Comment:
i need to remove this line because maybe my Mobile device do not support this...

Reply:
it absolutely makes sense, but please remove only this part:

` sourceWidth:1280; sourceHeight:960; displayWidth: 1280; displayHeight: 960`

Reply:
yes it's like that, i deleted only that piece of code i have only commented on the left side,

Reply:
Anyway it's strange because it's only the double of 640 x 480.